util/validator.py

```diff
@@ -0,0 +1,8 @@
+from lib.target import Target
+
+
+def validateTarget(host):
```

Comment:
We already have most of this functionality added to the Target object, what does making this another object serve?

Reply:
Will remove.